pkg/controller/service/service_controller.go

```diff
@@ -0,0 +1,125 @@
+package service
```

Comment:
A more specific name might be better?  router_service_controller.go?

Reply:
Done. I renamed to package, file names, and reconcile function.